### DISCUSSION TITLE
Importing TeslaFi data to TeslaMate

### DISCUSSION THREAD

**Discussion by Maasi1981** [teslamate-org/teslamate#1180] [General]
Unfortunately, I am not a hack/coding-expert - but I did manage to make TeslaMate working after several tryouts of different setup guides.

I do want to upload my exported TeslaFi data into TeslaMate and ran into a non-saying guide that I dont have a glue on.
It says add import to docker-compose.yml and so on.

But - in my head I am asking; what to do before and so on?

E.g.:

1) Login to your TeslaMate via SSH bla bla.
2) Find this, or do that with nano .something bla bla.

So - sorry, I would like to know step-by-step procedure to import TeslaFi data to TeslaMate - anyone who can help?

Thanks!

**Comment by larskochhansen** [comment]
I have the same "problem". I know how to get to the SSH but I do not know which command to use in order to copy my teslafi backup filer from my mac to the server via the SSH prompt.

I would also really love a step by step guide :-)

Thanks in advance 🤩
